test/suites/ACID.MATLAB.8.3.md5

```diff
@@ -40,7 +40,7 @@ legendsubplots : 528e57cc9c7858f9c941bd64733b2f12
 linesWithOutliers : 4976c3f7b1e8e8cd32f7e17e34fcc99f
 logbaseline : 59ff71a8a8125053e2ddac4dd3070e5a
```

Comment:
The hash changes in this PR are most likely easier to follow in the [commit-per-commit view](https://github.com/matlab2tikz/matlab2tikz/pull/759/commits).
For most plots, I just changed the test such that Octave 4.0 output matches Octave 3.8.
In other cases, that was impossible, so I had to change the hashes, but I did so after verifying graphically that it looks the same.

What @PeterPablo says is true:
 - different versions of MATLAB/Octave produce different output.

The same holds for different OSes and so on, but we currently try to factor those effects out as much as possible.